code/datums/diseases/transformation.dm

```diff
@@ -40,7 +41,7 @@
 	if(istype(affected_mob, /mob/living/carbon) && affected_mob.stat != DEAD)
 		if(stage5)
 			to_chat(affected_mob, pick(stage5))
-		if(jobban_isbanned(affected_mob, new_form))
+		if(jobban_isbanned(affected_mob, job_role))
 			affected_mob.death(1)
```

Comment:
Could you also change this to not have the `gibbed` argument? Currently it misses a lot of the standard 'death' things because of it: (Mob doesn't fall over, no deathgasp, etc.)
![gibbed](https://user-images.githubusercontent.com/57483089/136015833-f0e55560-3e2f-4515-8eb9-5171714732f7.PNG)

Without the `gibbed` argument:
![normal](https://user-images.githubusercontent.com/57483089/136016000-f777c2ac-ed6c-44a3-92db-792270172186.PNG)

This was in here already of course, but since you're making rest of the system actually work correctly it'd be good to do this part as well.
```suggestion
			affected_mob.death()
```